lib/renderers/0-2.js

```diff
@@ -128,18 +138,26 @@ export default class Renderer {
     let elements = [element];
     let currentElement = element;
 
+    let pushElement = (openedElement) => {
+      currentElement.appendChild(openedElement);
+      elements.push(openedElement);
+      currentElement = openedElement;
+    };
+
     for (let i=0, l=markers.length; i<l; i++) {
       let marker = markers[i];
       let [openTypes, closeCount, text] = marker;
 
       for (let j=0, m=openTypes.length; j<m; j++) {
         let markerType = this.markerTypes[openTypes[j]];
         let [tagName] = markerType;
-        if (isValidMarkerType(tagName)) {
+        let lowerCaseTagName = tagName.toLowerCase();
+        if (this.markupElementRenderer[lowerCaseTagName]) {
```

Comment:
Hm. Can we constrain this to only open an element (regardless of using a markupRenderer or not) if it is a valid type?

Seems more conservative of a change. I'm wary of allowing arbitrary marker types. At least it should be a different discussion than adding the markup renderers.